### PR TITLE
Updated process list tome output (#423)

### LINKED ISSUE #423: Improve the process list tome output
**Is your feature request related to a problem? Please describe.**
The process list tome output is kinda hard to read.
I really like process trees like with `ps -faux`

**Describe the solution you'd like**
Update the process list tome to print a process tree.
Similar to this:
```bash
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         729  0.0  0.0 241228  4864 ?        Ssl   2023   0:00 /usr/sbin/gdm3
root        1307  0.0  0.0 170240  5504 ?        Sl    2023   0:00  \_ gdm-session-worker [pam/gdm-launch-environment]
gdm         1402  0.0  0.0 162428  3200 tty1     Ssl+  2023   0:00      \_ /usr/libexec/gdm-x-session dbus-run-session -- gnome-session --autostart /usr/share/gdm/greeter/autostart
gdm         1407  0.0  0.0 1489800 8704 tty1     Sl+   2023   0:02          \_ /usr/lib/xorg/Xorg vt1 -displayfd 3 -auth /run/user/128/gdm/Xauthority -nolisten tcp -background none -noreset -keeptty -novtswitch -verbose 3
gdm         1516  0.0  0.0   6492  1152 tty1     S+    2023   0:00          \_ dbus-run-session -- gnome-session --autostart /usr/share/gdm/greeter/autostart
gdm         1517  0.0  0.0   8720  2816 tty1     S+    2023   0:00              \_ dbus-daemon --nofork --print-address 4 --session
root         719  0.0  0.0  15424  4352 ?        Ss    2023   0:00 sshd: /usr/sbin/sshd -D [listener] 0 of 10-100 startups
root      133960  0.0  0.0  17296 10880 ?        Ss   13:19   0:00  \_ sshd: sysadmin [priv]
sysadmin  134038  0.0  0.0  17432  7864 ?        S    13:19   0:00      \_ sshd: sysadmin@pts/0
sysadmin  134039  0.0  0.0  11276  5248 pts/0    Ss   13:19   0:00          \_ -bash
root      134129  0.0  0.0  14348  5760 pts/0    S+   13:19   0:00              \_ sudo su -
root      134130  0.0  0.0  14348  2480 pts/5    Ss   13:19   0:00                  \_ sudo su -
sysadmin   64686  0.0  0.0  14664  5516 ?        Ss    2023   0:05 tmux
sysadmin   64687  0.0  0.0  11300  4352 pts/1    Ss    2023   0:00  \_ -bash
sysadmin  122694  0.0  0.0 1550072 19432 pts/1   Sl+   2023   0:07  |   \_ docker run -v /home/sysadmin/Documents/projects/:/projects/ -it rustc-ollvm:latest /bin/bash
sysadmin   65167  0.0  0.0  11432  4224 pts/2    Ss+   2023   0:00  \_ -bash
```

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by arunjohnkuruvilla**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

Update the process list tome for the output to look like the following -

```bash
PID             PPID            username                        command
53              1               root                            /lib/systemd/systemd-udevd
428             354             ubuntu                          -bash
```

#### Which issue(s) this PR fixes:

Hulto: Gonna change 423 to different tome editing so the issue stays open.

### CODE REVIEW

**Review by hulto — DISMISSED** [review]
Hey @arunjohnkuruvilla!
Thanks for the feature. This looks really nice 🤌
Ran locally everything looks good.

When you get a chance can fill out this feedback form so we can improve the developer experience:
https://forms.gle/3Rn3iQqF79B68nWJ7

**Comment by hulto** [comment]
One change that would be nice to have (which we is an existing bug)
On line 36
Add a call to replace. Sometimes the command can include newlines and escaping them will keep the output cleaner.
```python
            print(current_proc_command+"\n")
...
            print(current_proc_command.replace("\n","\\n")+"\n")
```

File: tavern/tomes/process_list/main.eldritch [hunk 1]
```diff
@@ -1,14 +1,39 @@
...
+                current_proc_command = proc['name']
+
+            current_proc_pid = str(proc['pid'])
+            current_proc_ppid = str(proc['ppid'])
+            current_proc_username = proc['username']
+
+            print(pad_pid(current_proc_pid))
+            print(pad_pid(current_proc_ppid))
+            print(pad_username(current_proc_username))
+            print(current_proc_command+"\n")
```
Comment (hulto) [inline tavern/tomes/process_list/main.eldritch:36]: Small change
```python
print(current_proc_command.replace("\n","\\n")+"\n")
```

Comment (arunjohnkuruvilla) [reply tavern/tomes/process_list/main.eldritch]: Pushed the change in the following commit.

**Review by hulto — APPROVED** [review]
LGTM :shipit:

**Comment by hulto** [comment]
Incredible feedback @arunjohnkuruvilla 
Very fair there are no docs on tome creation i've opened #445 to resolve this for future contributors.

We do have a way for you to test tomes without restarting the application (however it's not documented yet 😬 )
You can use the golem standalone interpreter either interactively or with a specified tome.

```bash
# Run eldritch interactively - I find this useful for testing specific syntax or experimenting with dictionaries.
[realm/implants/golem]$ cargo run -- -i
# Run a specific tome - I statically set any input params in the tome first to resolve run time errors.
[realm/implants/golem]$ cargo run -- 'realm/tavern/tomes/process_list/main.eldritch'
```

I'll include these ^ in the docs update but hope this helps.

**Comment by Cictrone** [comment]
Merging because it’s awesome :D

**Pull request merged**